pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -142,8 +156,10 @@ func (pl *TaintToleration) Score(ctx context.Context, state *framework.CycleStat
 func (pl *TaintToleration) NormalizeScore(ctx context.Context, _ *framework.CycleState, pod *v1.Pod, scores framework.NodeScoreList) *framework.Status {
 	// Note that ComputeTaintTolerationPriorityReduce doesn't use priority metadata, hence passing nil here.
 	normalizeFun := priorities.NormalizeReduce(framework.MaxNodeScore, true)
```

Comment:
We could use DefaultNormalizeScore in helper package

Reply:
done.